tests/test_utils.py

```diff
@@ -0,0 +1,131 @@
+from openapi_server.apis.utils import error_handling
+from fastapi.exceptions import HTTPException
+from caselawclient.Client import (
+    MarklogicUnauthorizedError,
+    MarklogicValidationFailedError,
+)
+from io import StringIO
+from unittest.mock import Mock
+import sys
+
+
+def test_error_handling_no_exception():
+    def example():
+        with error_handling():
+            return 4
+
+    assert example() == 4
+
+
+def test_error_handling_python_error():
+    """
+    Given you will get a standard Python exception
+    When using error_handling
+    Then the error response:
+      contains a very generic error with no contents
+      has a relevant status code
+    And outputs the message and traceback to the logs
+    """
+
+    def example():
+        with error_handling():
+            1 / 0
+
+    output = StringIO()
+    error = StringIO()
+    sys.stdout = output
+    sys.stderr = error
+    try:
+        example()
+    except HTTPException as ex:
+        sys.stdout = sys.__stdout__
+        sys.stderr = sys.__stderr__
+
+        assert ex.detail == "An unknown error occurred outside of Marklogic."
+        assert ex.status_code == 500
+        assert "division by zero" in error.getvalue()
+        assert "1 / 0" in error.getvalue()
+    else:
```

Comment:
This test isn't parametrized so I don't see how we can hit both the except and else blocks, unless I am missing something?

Either way, if these are two scenarios we want to test, it could be worth to split them into two separate tests so each test is only concerned with one action so that debugging them later is simpler.